impl/src/main/java/org/jboss/cdi/tck/tests/full/interceptors/contract/invocationContext/ProducerBean.java

```diff
@@ -0,0 +1,19 @@
+package org.jboss.cdi.tck.tests.full.interceptors.contract.invocationContext;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.inject.Produces;
+import jakarta.enterprise.inject.spi.InterceptionFactory;
+
+@ApplicationScoped
+public class ProducerBean {
+
+    @Produces
+    @ApplicationScoped
+    public Product createProduct(InterceptionFactory<Product> interceptionFactory) {
+        interceptionFactory.configure().add(ProductInterceptorBinding1.Literal.INSTANCE).filterMethods(m -> m.getJavaMember().getName().equals("ping"))
+                .findFirst().get()
+                .add(ProductInterceptorBinding2.Literal.INSTANCE)
+                .add(ProductInterceptorBinding3.Literal.INSTANCE);
```

Comment:
Wouldn't it be better to add only `ProductInterceptorBinding2` to `ping` and have another method to which we would add `ProductInterceptorBinding3`?

Reply:
@Ladicek I've updated the PR to use two different interceptors and two methods that have both different sets of bindings

Reply:
Thanks!